packages/loader/container-loader/src/prefetchDocumentStorageService.ts

```diff
@@ -36,18 +40,18 @@ export class PrefetchDocumentStorageService extends DocumentStorageServiceProxy
         this.prefetchCache.clear();
     }
 
-    // eslint-disable-next-line @typescript-eslint/promise-function-async
-    private cachedRead(blobId: string): Promise<string> {
+    private async cachedRead(blobId: string): Promise<ArrayBufferLike> {
+        const blob = await this.internalStorageService.readBlob(blobId);
```

Comment:
The purpose of this class is to prefetch blobs and return them from cache on next reads.
Moving read up and making it unconditional defeats the purpose of this class.
Could you please keep the structure as it was before?